deploy/infra/lib/constructs/targetgroup-granter.ts

```diff
@@ -7,11 +7,13 @@ import * as path from "path";
 import { EventBus } from "aws-cdk-lib/aws-events";
 import { grantAssumeHandlerRole } from "../helpers/permissions";
 import { Table } from "aws-cdk-lib/aws-dynamodb";
+import * as ec2 from 'aws-cdk-lib/aws-ec2'
```

Comment:
```suggestion
import { Vpc } as ec2 from 'aws-cdk-lib/aws-ec2'
```
To follow the pattern